README.md

```diff
@@ -1 +1,68 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# 📖 Hello Microverse
+
+> •	Hello Microverse is a simple project. It is the first project created as part of Microverse curriculum.
+
+- _The main page shows a text "Hello Microverse!"._
+  ­
+
+## Built With
+
+- HTML & CSS
+- [Linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js)
+
+## Video Description
+
+View [video presentaton](https://www.awesomescreenshot.com/video/15817834?key=6aa71e50744b09954c38df0e778377a3)
+
+## Getting Started
+
+### Prerequisites
+
+- node.js (10.13.0 (LTS))
+- git
+
+### Setup
+
+To get a local copy, follow the steps below:
+
+1. Open your terminal
+2. Navigate to the directory where you want to clone the project.
+3. Clone the project using `git clone` or directly by running the following command
+
+   `git clone git@github.com:iamsjunaid/Hello-Microverse.git`
+
+4. Open your editor and follow the steps mentioned below
+
+### Install
+
+- `npm install`
+
+### Usage
+
+- `npm run build`
+- `npm start`
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@iamsjunaid](https://github.com/iamsjunaid)
+- Twitter: [@junaidahmedsd](https://twitter.com/junaidahmedsd)
+- LinkedIn: [Junaid Ahmed](https://www.linkedin.com/in/junaid-syed-058b2779)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/iamsjunaid/Hello-Microvese/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [Optional] Always make sure you don't have any broken links in your Readme. In this case, the license file is not present in your repository.